src/package-set.coffee

```diff
@@ -0,0 +1,11 @@
+Q = require 'q'
```

Comment:
Why not use the native promises? This line is redundant.
SideNote: You are requiring the variable `Q` and doing `Promise.resolve`, so it already uses native ones